### PR TITLE
Greek

### PR DESCRIPTION
**Pull request by ippocratis**
Translated strings.xml to Greek
not sure what to do with plurals
Should I translate the words zero,one,other?

### CODE REVIEW

**Comment by ippocratis** [comment]
Nevermind
I think I got it

**Comment by Radiokot** [comment]
If you are not familiar with the Android plurals system, please read [this explanation and check out the example](https://developer.android.com/guide/topics/resources/string-resource#Plurals). In some languages there are just two options, for one and many, but in other there may be more. If it is for Greek, please fill in the required options.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,121 +1,121 @@
...
+    <string name="loading_albums_progress">φόρτωση άλμπουμ…</string>
+    <string name="reload_albums">επαναφόρτωση άλμπουμ</string>
+    <string name="no_albums_found">δε βρέθηκαν άλμπουμ</string>
+    <string name="clear_selection">καθαρισμος επιλογής</string>
+    <string name="done_selecting">ολοκλήρωση</string>
     <string name="template_current_of">%1$d of %2$d</string>
-    <string name="search_doesnt_fit_allowed_types">Search query doesn\'t fit the allowed types</string>
-    <string name="save_file_to_the_storage">Save the file to the storage</string>
-    <string name="share_the_file">Share the file</string>
+    <string name="search_doesnt_fit_allowed_types">ο όρος αναζήτησης δεν\ταυτίζεται με τους επιτρεπόμενους τύπους</string>
```
Comment (Radiokot) [inline app/src/main/res/values/strings.xml:118]: The `\` in the strings is for escaping `'`. If there is no `'` or other XML special symbols in your string, you can remove `\` as well.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,121 +1,121 @@
...
-    <string name="learn_more">Learn more</string>
-    <string name="preferences">Preferences</string>
-    <string name="library_preferences">@string/library</string>
-    <string name="about">About</string>
-    <string name="app_version">Version</string>
-    <string name="report_an_issue">Report an issue</string>
-    <string name="template_error_failed_to_export_bookmarks">Failed to export bookmarks: %1$s</string>
+    <string name="learn_more">μάθε περισσότερα</string>
+    <string name="preferences">προτιμήσεις</string>
+    <string name="library_preferences">@string/προτιμήσεις</string>
```
Comment (Radiokot) [inline app/src/main/res/values/strings.xml:89]: Just keep a reference to `@string/library` for this value. This reference means that the library preferences title is the same string as "library", defined above. The current value is invalid.

**Review by Radiokot — CHANGES_REQUESTED** [review]
Please, check out the comments.
Also, put the translated files to the `values-el` directory and restore the default (English) files in the `values` directory.

**Pull request closed without merging**